Buildings/Experimental/DHC/Networks/Distribution1PipeAutoSize.mo

```diff
@@ -1,56 +1,51 @@
-within Buildings.Experimental.DHC.Networks.Combined;
-model UnidirectionalSeries
-  "Hydronic network for unidirectional series DHC system"
-  extends Buildings.Experimental.DHC.Networks.BaseClasses.PartialDistribution1Pipe(
+within Buildings.Experimental.DHC.Networks;
+model Distribution1PipeAutoSize
+  "Model of a one-pipe distribution network with auto-size pipes in the main line"
+  extends
+    Buildings.Experimental.DHC.Networks.BaseClasses.PartialDistribution1Pipe(
     tau=5*60,
-    redeclare Buildings.Experimental.DHC.Networks.Combined.BaseClasses.ConnectionSeriesAutosize con[nCon](
+    redeclare
+      Buildings.Experimental.DHC.Networks.Connections.Connection1PipeAutosize
+      con[nCon](
       each final dp_length_nominal=dp_length_nominal,
       final lDis=lDis,
-      final lCon=lCon,
-      final dhDis=dhDis,
-      final dhCon=dhCon),
+      final dhDis=dhDis),
     redeclare model Model_pipDis =
-        Buildings.Experimental.DHC.Networks.Combined.BaseClasses.PipeAutosize
-        (
-      roughness=7e-6,
-      fac=1.5,
-      final dp_length_nominal=dp_length_nominal,
-      final dh(fixed=true)=dhEnd,
-      final length=lEnd));
+        Buildings.Experimental.DHC.Networks.Pipes.PipeAutosize (
+        final dp_length_nominal=dp_length_nominal,
+        final dh(fixed=true) = dhEnd,
+        final length=lEnd),
+    pipEnd(fac=1));
   parameter Real dp_length_nominal(final unit="Pa/m") = 250
     "Pressure drop per pipe length at nominal flow rate";
   parameter Modelica.Units.SI.Length lDis[nCon]
     "Length of the distribution pipe before each connection";
-  parameter Modelica.Units.SI.Length lCon[nCon]
-    "Length of each connection pipe (supply only, not counting return line)";
+
   parameter Modelica.Units.SI.Length lEnd
     "Length of the end of the distribution line (after last connection)";
-  parameter Modelica.Units.SI.Length dhDis[nCon](
+  final parameter Modelica.Units.SI.Length dhDis[nCon](
     each fixed=false,
     each start=0.05,
     each min=0.01)
     "Hydraulic diameter of the distribution pipe before each connection";
-  parameter Modelica.Units.SI.Length dhCon[nCon](
-    each fixed=false,
-    each start=0.05,
-    each min=0.01) "Hydraulic diameter of each connection pipe";
-  parameter Modelica.Units.SI.Length dhEnd(
+
+  final parameter Modelica.Units.SI.Length dhEnd(
     fixed=false,
     start=0.05,
     min=0.01)
     "Hydraulic diameter of of the end of the distribution line (after last connection)";
   annotation (Documentation(info="<html>
 <p>
-This model represents a one-pipe distribution network with built-in computation
-of the pipe diameter based on the pressure drop per pipe length
-at nominal flow rate.
+This is a model of a one-pipe distribution network using a connection model with an auto-sized pipe in the main line whose hydraulic diameter 
```

Comment:
Add link to connection model used so user can quickly refer to it.  Same comment for all 4 Distribution models.

Reply:
Added connection link to all distribution models